README.md

```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
+    - For creating Boards subscriptions
 
     ```
-    - /azuredevops boards subscribe
+    - /azuredevops boards subscription add
     ```
 
-- View/List subscriptions: A user can view the list of subscriptions for a project by going on the subscriptions list page after clicking on the project title under "Linked Projects". Users can also view the list of all subscriptions for a channel by using the below slash command in the channel.
+    - For creating Repos subscriptions
 
     ```
-    - /azuredevops boards subscriptions
+    - /azuredevops repos subscription add
     ```
 
-- Delete subscriptions: A user can delete subscriptions for a project from RHS by going on the subscriptions list page after clicking on the project title under "Linked Projects". Users can also delete a subscription for a project by using the slash command below.
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only create/delete a subscription.
+
+- View/List subscriptions: A user can view the list of subscriptions for a project by going to the subscriptions list page after clicking on the project title under "Linked Projects". Users can also view the list of all subscriptions for a channel by using the below slash command in the channel.
+
+    - For listing Boards subscriptions
+
+    ```
+    - /azuredevops boards subscription list anyone all_channels
+    ```
+
+    - For listing Repos subscriptions
+
+    ```
+    - /azuredevops repos subscription list anyone all_channels 
+    ```
+
+    Supported filters on the above slash command:
+    - CreatedBy: `me`(show all subscriptions created by the current Mattermost user), `anyone`(show all subscriptions created by any Mattermost user)
+    - Show for all channels: when the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed, you can leave this to list the subscriptions of the current channel only.
+
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only view/list subscriptions that exist in a channel where he/she is not a member.
+
+- Delete subscriptions: A user can delete subscriptions for a project from RHS by going to the subscriptions list page after clicking on the project title under "Linked Projects". Users can also delete a subscription for a project by using the slash command below.
+
+    - For deleting Boards subscriptions
 
     ```
-    - /azuredevops boards unsubscribe [subscription id]
+    - /azuredevops boards subscription delete [subscription id]
     ```
 
+    - For deleting Repos subscriptions
+
+    ```
+    - /azuredevops repos subscription delete [subscription id]
+    ```
+
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only create/delete a subscription.
```

Comment:
same

Reply:
yes this also holds true only Admins can create/delete subscription